packages/core/src/polyfills/aria-reflect.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+/*
+ * Shim for aria atribute reflection missing in Firefox
+ * https://wicg.github.io/aom/aria-reflection-explainer.html
+ * https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaLabel
+ * Patching missing types as needed
+ */
+declare global {
+  interface Element {
+    role: string | null;
+    // ARIAMixin interface in TS lib.dom.d.ts https://github.com/microsoft/TypeScript/issues/46456
+    ariaActiveDescendant: string | null;
+    ariaControls: string | null;
+    ariaLabelledBy: string | null;
+    ariaDisabled: string | null;
+    ariaExpanded: string | null;
+    ariaSelected: string | null;
+  }
+}
```

Comment:
This augments some of the aria properties as nullable. TypeScript marks them as string only but this is incorrect and causing issues when trying to set them as null. Once this is fixed in TS we can remove the overloaded `Element` interface.

 This is impacting other Design systems as well https://github.com/microsoft/TypeScript/issues/46456